target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -535,9 +538,12 @@ ramips_setup_macs()
 		wan_mac=$(macaddr_add "$lan_mac" 1)
 		;;
 	asus,rt-ac57u|\
+	elecom,wrc-1167ghbk2-s|\
+	elecom,wrc-1900gst|\
+	elecom,wrc-2533gst|\
 	phicomm,k2p|\
-	planex,vr500)
-		lan_mac=$(mtd_get_mac_binary factory 0xe000)
+	planex,vr500|\
```

Comment:
Have you confirmed that this router has a wan_mac stored at 0xe006?

Reply:
No, I haven't. This patch is only about moving code without changing the behavior I found in this file.
So, I do not know whether the wan_mac is right or wrong, but I do not have to since I'm not changing it.

Changes based on actual tests/confirmations are only implemented in "ramips: initialize MAC addresses from flash where possible" and Newifi Y1(s) patch

Reply:
Oh. I misread the patch a bit :)

Reply:
Yes, for changes like this the git diff is quite misleading sometimes.